Makefile

```diff
@@ -75,6 +75,16 @@ docker-toolchain:
 		echo "Repo is dirty, commit first." ;\
 	fi
 
+docker-rust-toolchain:
```

Comment:
yeah sorry. I was really lazy with this. For the tag name, it's because otherwise the sed in update deps would have trampled over the `-rust-` bit in the circleci config (it replaces `toolchain-*` with `toolchain-HASH`, which means `toolchain-rust-` got replaced. I can just fix the sed.